tokio/src/runtime/time/entry.rs

```diff
@@ -334,10 +334,14 @@ pub(crate) struct TimerShared {
     /// Only accessed under the entry lock.
     pointers: linked_list::Pointers<TimerShared>,
 
-    /// The expiration time for which this entry is currently registered.
+    /// The time when the [`TimerEntry`] was registered into the Wheel,
+    /// [`STATE_DEREGISTERED`] means it is not registered.
+    ///
     /// Generally owned by the driver, but is accessed by the entry when not
-    /// registered.
-    cached_when: AtomicU64,
+    /// registered, but we only need [`Ordering::Relaxed`],
+    /// please check out the module-level documentation for
+    /// the safety explanation.
```

Comment:
The wording of this note about relaxed confuses me. What do you mean here?

Reply:
Confusing note has been fixed.